wavefront/server/modules/triggers_module/triggers_module/providers/gmail/gmail_provider.py

```
@@ -0,0 +1,473 @@
+            if binding is None:
+                policy.bindings.add(role=wants_role, members=[wants_member])
+                mutated = True
+            elif wants_member not in binding.members:
+                binding.members.append(wants_member)
+                mutated = True
+            if mutated:
+                publisher.set_iam_policy(
+                    request={'resource': topic_path, 'policy': policy},
+                    timeout=30,
+                )
+        except Exception as exc:
```

Ruff (0.15.13)

[warning] Do not catch blind exception: `Exception`

(BLE001)